### DISCUSSION TITLE
Gantt chart for Timeline

### DISCUSSION THREAD

**Discussion by salah-walid** [iTwin/itwinjs-core#3746] [Q&A] [answered]
Hi,

How to achieve the same effect as the synchro control page, where there is a gantt chart at the bottom when there is a timeline
![image](https://user-images.githubusercontent.com/29980658/171872877-db1e7b29-3662-4616-8f81-dffaec723ee6.png)

**Comment by pmconne** [comment, marked as answer]
The chart in Synchro Control is produced using [DHTMLX JS Gantt library](https://dhtmlx.com/docs/products/dhtmlxGantt/). Unfortunately we cannot publish the control for use by other developers, because that library requires a paid license.

**Reply by salah-walid** [reply]
Thanks, My next question is, how to get the timeline info used to create it ?, maybe i can display the chart using another library

**Reply by pmconne** [reply]
[viewport.displayStyle.scheduleScript](https://www.itwinjs.org/reference/core-frontend/views/displaystylestate/#schedulescript) will provide you the [schedule script](https://www.itwinjs.org/reference/core-common/displaystyles/renderschedule/renderschedule.script/) defining the timeline.

**Reply by salah-walid** [reply]
I printed it's value, and the closest property resembling what i need is
``` modelTimeLines.elementTimeLines[index] ```
![image](https://user-images.githubusercontent.com/29980658/171909535-42cfe895-632a-4566-a0e2-5f9eb302f3ab.png)
But i don't see a node name for example.

**Reply by pmconne** [reply]
That's all the data that gets stored in the iModel - no task names, no Ids. They must be pulling additional schedule data from elsewhere.
Note that it's called a "RenderSchedule" script because it's about animating the view according to the schedule - it is not intended as a complete representation of the work schedule.
